index.html

```diff
@@ -1355,6 +1367,183 @@ <h4>Exchange Examples</h4>
+      <figure>
+        <pre class="mermaid">
+sequenceDiagram
+  participant H as Holder
+  participant W as Holder Coordinator (Wallet)
+  participant V as Verifier Coordinator
+  autonumber
+  H->>W: Tap "Share credential"
+  W->>W: Generate Interaction URL
+  W->>W: Display QR Code
+  H->>V: Present QR Code
+  V->>V: Scan QR Code and decode URL
+  V->>W: Get available interaction protocols
+  Note left of V: GET &lt;https://wallet.example/interactions/&lt;interactionId>?wi_api=&lt;api_version>
+  W->>V: Return interaction protocols
+  Note left of V: (VC API, OID4, NFC, Bluetooth, etc.)
+  V->>V: Select appropriate protocol
+  V->>W: Transmit protocol selection and options
+  W->>H: Request consent to proceed
+  Note right of H: "Open browser to verifier.example?"
+  H->>W: Confirm consent start exchange
+  W->>V: Start exchange
+        </pre>
+        <figcaption>
+A Holder-initiated exchange using a QR Code
+        </figcaption>
+      </figure>
+
+      <p class="note" title="Multiple interaction initiation variations are possible">
+While the example above is a [=holder=]-initiated interaction, [=issuers=] and
+[=verifiers=] are also capable of initiating an interaction. Similarly, while
+the transmission of the interaction URL is performed via a QR Code
+(optical medium), the interaction URL can also be transmitted over NFC or
+Bluetooth (wireless medium).
+      </p>
+
+      <section>
+        <h4>Interaction URL Format</h4>
+        <p>
+The format for the <dfn>interaction URL</dfn> MUST conform to the syntax for the
+[[[URL]]]. The interaction URL SHOULD be an HTTPS URL where the path segment
+ends with the `/interactions/` path, a transaction-specific `interactionId`, and
+the interaction URL version number which MUST be `iuv=1`. The URL SHOULD be
+opaque and require no URL syntax processing before it is fetched by the
+receiving system.
+        </p>
+        <p>
+When the [=interaction URL=] is fetched using an `Accept` header of
+`application/json`, a single JSON object containing a `protocols` [=map=] MUST
+be returned.
+        </p>
+        <p>
+When the [=interaction URL=] is fetched using any unrecognized `Accept`
+header, an `text/html` document MUST be returned with instructions to scan the
+URL using software that understands how to process interaction URLs.
+        </p>
+      </section>
+
+      <section>
+        <h4>Interaction QR Code Format</h4>
+        <p>
+The format for an <dfn>interaction QR Code</dfn> MUST be an [=interaction URL=]
+expressed as a QR code according to [[[ISO18004]]]. To ensure broad
+interoperability, the length of the [=interaction URL=] SHOULD be as short as
+possible, SHOULD NOT exceed 400 alphanumeric characters, and MUST NOT exceed
+4,296 alphanumeric characters. An example of an interaction QR code can be found
+below:
+        </p>
+
+        <figure id="roles">
+          <img style="margin: auto; width: 33%; display: block;"
+         src="./diagrams/interaction-qr-code.png"
```

Comment:
Missing from PR.

Reply:
Added in https://github.com/w3c-ccg/vc-api/pull/473/commits/4675a21f1d9279c63892ab080d2d83ef3d166fe3.